orc8r/cloud/test/scale/basic_test.go

```diff
@@ -0,0 +1,429 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package scale_test
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"magma/orc8r/cloud/go/obsidian/swagger/v1/client"
+	"magma/orc8r/cloud/go/obsidian/swagger/v1/client/a_p_ns"
+	"magma/orc8r/cloud/go/obsidian/swagger/v1/client/lte_gateways"
+	"magma/orc8r/cloud/go/obsidian/swagger/v1/client/lte_networks"
+	"magma/orc8r/cloud/go/obsidian/swagger/v1/client/subscribers"
+	"magma/orc8r/cloud/go/obsidian/swagger/v1/client/upgrades"
+	"magma/orc8r/cloud/go/obsidian/swagger/v1/models"
+	"magma/orc8r/cloud/test/testlib"
+
+	oclient "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/swag"
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/api"
+	prom "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/crypto/pkcs12"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+// TODO(hcgatewood): clean up this scale test then add it to daily CI job
+
+func init() {
+	_ = flag.Set("alsologtostderr", "true")
+}
+
+var NSubs int64 = 20_000
+var NGateways = 198
+
+// var HardwareID = "b639e30b-1fd4-4f45-934a-9ce44a1cf880"
```

Comment:
Do we want these committed?

Reply:
Yeah it's still a WIP so I think it's fine to leave it committed